src/devices_models/devices/common/add_variable.jl

```diff
@@ -1,3 +1,134 @@
+struct AddVariableInputs
+    variable_names::Vector{String}
```

Comment:
Normally we expect only a single variable. We should make a constructor that takes just one variable. Only in the case of Storage and commitment we have more than one variable at a time.